src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarTemplateSettings.cs

```diff
@@ -14,10 +14,10 @@ public AppBarTemplateSettings(AppBar appBar)
 		// TODO: Implement
 		public Rect ClipRect => new Rect(0, 0, double.PositiveInfinity, double.PositiveInfinity); // No clipping
 		public Thickness CompactRootMargin => new Thickness(0);
-		public double CompactVerticalDelta => 0;
+		public double CompactVerticalDelta => -16;
```

Comment:
Where do those magic numbers come from? :) We probably want comments documenting them.

Reply:
From windows, but even then, they may be computed values and we would not know it.